main.js

```diff
@@ -30,5 +31,7 @@ client.on('ready', () => {
         }
     });
 });
-
+client.on('message', message => { 
+    mentionHandler();
+});
```

Comment:
czy zamiast tego nie powinno być po prostu tak?
`
client.on('message", mentionHandler);
`
mentionHandler powinien dostawać jako argument message, jak to działa skoro nie podajesz mu wiadomości do sprawdzenia?